packages/x-components/tests/e2e/cucumber/recommendations/recommendations.spec.ts

```diff
@@ -1,24 +1,21 @@
 import { And, Given, Then } from 'cypress-cucumber-preprocessor/steps';
-import { InstallXOptions } from '../../../../src/x-installer/x-installer/types';
-import { TopRecommendationsRequest } from '../../../../../search-adapter/src/types/requests.types';
+import { TopRecommendationsRequest } from '@empathyco/x-adapter';
+import { recommendationsStub } from './stubs/recommendations.stub';
 
-let config: InstallXOptions['xModules'];
+Given('suggestions API should respond with mocked suggestions', () => {
+  cy.intercept('https://api.empathy.co/getTopRecommendations', req => {
+    req.reply(recommendationsStub);
+  }).as('interceptedRecommendations');
+});
 
-Given('following config: max items to store is {int}', (maxItemsToRequest: number) => {
-  config = {
+And('following config: max items to store is {int}', (maxItemsToRequest: number) => {
+  const config = {
```

Comment:
Never define constant objects without a type. This is a pain point when we do refactors becase neither TS or us knows what type this object should match. In this case it should be `InstallXOptions['xModules']`